src/dynafed.rs

```diff
@@ -24,7 +24,7 @@ use encode::{self, Encodable, Decodable};
 use Script;
 
 /// Dynamic federations paramaters, as encoded in a block header
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+#[derive(Clone, Eq, Hash, PartialEq)]
```

Comment:
I am curious why did not derive Ord/PartialOrd because I was convinced by your reasoning here(https://github.com/rust-bitcoin/rust-bitcoin/pull/524#issuecomment-734386157)

Reply:
done